README.md

```diff
@@ -134,14 +135,14 @@ To build the app for production and start the server run:
 ```bash
 yarn start
 ```
+This will run the application in an instance of the Express server, which is included.
 
 If you only want to build for production, without starting, run:
 
 ```bash
 yarn run build:prod
 ```
-
-This will build the application and put the result in the `dist` folder
+This will build the application and put the result in the `dist` folder.  You can copy this folder to wherever you need it for your application server.  If you will be using the built-in Express server, you'll also need a copy of the `node_modules` folder tucked inside your copy of `dist`.
```

Comment:
I'm not sure I understand the second sentence here (though I've not tried using `yarn run build:prod` myself)...could we maybe use an example here to better describe what "need(ing) a copy of the `node_modules` folder" means?

Or, to reword that request... what I'd expect in these docs are an example of what to do *after* running `yarn run build:prod`.  Could we provide a basic example here by showing how you'd use the built-in Express server to run what is in `dist` and in `node_modules`?  Is it easy enough to show the steps here?

Reply:
Mark is suggesting that after you run `yarn run build:prod` you can move the `dist` folder it generates anywhere, e.g. to a different server. As long as the same version of node is installed you can run it, provided you copy the `node_modules` folder as well and put it _inside_ that `dist` folder.

This was surprising to me, because I expected that the `node_modules` folder would have to be _outside_ the `dist` folder the way it is in the project itself. But I tested it, and it works if you move it inside as well. 
And if both work, mark's way is neater.

I did find that you have to start the server one level up from the dist folder: you have to run 
```bash
node dist/server.js
```

The following won't work at the moment:
```bash
cd dist
node server.js
```

We likely could make it work if that's preferred, but some relative paths assume the dist folder is inside the working dir at the moment.